documentation/modules/deploying/proc-deploy-setup-external-clients.adoc

```diff
@@ -80,21 +83,21 @@ metadata:
 spec:
   kafka:
     # ...
-    listeners: <1>
-    - name: external <2>
+    listeners: # <1>
+    - name: external # <2>
       port: 9094 <3>
-      type: _LISTENER-TYPE_ <4>
-      tls: true <5>
+      type: _<listener_type>_ # <4>
+      tls: true # <5>
       authentication:
-        type: tls <6>
+        type: tls # <6>
       configuration:
-        preferredNodePortAddressType: InternalDNS <7>
-        _bootstrap and broker service overrides_ <8>
+        preferredNodePortAddressType: InternalDNS # <7>
+        _bootstrap and broker service overrides_ # <8>
```

Comment:
Maybe we should keep it simple and remove this? Since the listener `type` is a placeholder, it is not even clear if these options make any sense.

Reply:
Makes sense. We can leave a callout to say that additional configuration options are available:

> <7> (Optional) Depending on the requirements of the listener type, you can specify additional link:{BookURLUsing}#type-GenericKafkaListenerConfiguration-reference[listener configuration^].